src/client/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -43,9 +43,9 @@ class ErrorableTextInput extends React.Component {
     let errorSpanId = undefined;
     let inputErrorClass = undefined;
     let labelErrorClass = undefined;
-    if (this.props.errorMessage) {
+    if (this.props.errorMessage || this.props.field.value.length > this.props.charMax) {
```

Comment:
Hm I didn't really think this through... I liked how you used the `maxLength` attribute, but I didn't realize that the message will never show up if we use it, hence why you removed that. Sorry! I'm rethinking this now. Perhaps the small text you had below the field is the best way to go, but just show it conditionally if `this.props.field.value.length >= this.props.charMax` so it'll only show up once they hit 30 characters, for example. Thoughts on that?